EpiAware/src/utilities.jl

```diff
@@ -28,12 +28,14 @@ function scan(f, init, xs)
 end
 
 """
-    create_discrete_pmf(dist; Δd = 1.0, D)
+    create_discrete_pmf(dist::Distribution, ::Val{:basic}; Δd = 1.0, D)
 
-Create a discrete probability mass function (PMF) from a given distribution.
+Create a discrete probability mass function (PMF) from a given distribution, assuming that the
```

Comment:
I may be misunderstanding, but isn't this implementation the simple single censoring case? With the assumption that we assign to the floor of the bin rather than the center?

Either way, would you mind adding some more clarification to the docstring about what this corresponds to re: single vs double censoring and "traditional" implementations?

Reply:
Yes, thats the one. As mentioned above, the main reason I dispatch with a Val type is that I don't want an easy option here; you have to select this version quite hard.

Reply:
Coolio -- if you could say that in the function documentation that would be helpful

Reply:
I've added a bit more description and slightly extended so its default mid-point approximation for the primary event (but flexible to be any point).